src/formatDate.js

```diff
@@ -50,7 +51,72 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateValue = {
+    day: 0,
+    month: 0,
+    year: 0,
+  };
+
+  const newDate = [];
+  const oldDate = date.split(fromFormat[3]);
+
+  for (let i = 0; i < fromFormat.length; i++) {
+    switch (fromFormat[i]) {
+      case 'DD':
+        dateValue.day = oldDate[i];
+        break;
+
+      case 'MM':
+        dateValue.month = oldDate[i];
+        break;
+
+      case 'YY':
+      case 'YYYY':
+        dateValue.year = oldDate[i];
+        break;
+
+      default:
+        break;
+    }
+  }
+
+  for (let i = 0; i < toFormat.length; i++) {
+    switch (toFormat[i]) {
+      case 'DD':
+        newDate.push(dateValue.day);
+        break;
+      case 'MM':
+        newDate.push(dateValue.month);
+        break;
+      case 'YY':
+        if (dateValue.year.length === 4) {
+          const y = dateValue.year[2] + '' + dateValue.year[3];
+
+          newDate.push(y);
```

Comment:
```suggestion
          newDate.push(dateValue.year.slice(2));
```
don't use variables names like `y`, `x` and so on